apps/desktop/src/components/v3/IntegrateUpstreamModal.svelte

```diff
@@ -170,23 +163,20 @@
 	}
 
 	async function fetchAppliedBranches() {
-		return await stackService
-			.fetchStacks(projectId)
-			.then(
-				(stacksResponse) =>
-					stacksResponse.data?.flatMap((stack) => stack.heads.map((head) => head.name)) ?? []
-			);
+		const stacksResponse = await stackService.fetchStacks(projectId);
+		return stacksResponse.data?.flatMap((stack) => stack.heads.map((head) => head.name)) ?? [];
```

Comment:
This change here is a no-op, it's the same thing just expressed using async/await syntax